src/sandstorm/supervisor.capnp

```diff
@@ -274,15 +256,28 @@ struct ApiTokenOwner {
 
       introducerIdentity @9 :Text;
       # The identity ID through which a user's powerbox action caused the grain to receive this
-      # token. This is the identity against which the `requiredPermissions` parameter
-      # to `restore()` will be checked. This field is only intended to be filled in by the
-      # front-end during a powerbox request; a regular `save()` call produces a capability that
-      # has no "introducer".
+      # token. The token will become invalid if this identity loses access to the grain. This field
```

Comment:
@dwrensha The problem is existing tokens were not only expecting `introducerIdentity` to have basic access but were also expecting to check it against some required permissions specified at restore() time. The new interface doesn't give any way to specify said required permissions. So, you are loosening the requirements here either way. Why is it OK to loosen them somewhat but not completely?

I think the only self-consistent options here are:
1. Support exactly what we had before, including keeping a way to specify requiredPermissions on restore.
2. Declare that because no one was actually using these yet (except feature-key-vendor), we don't care. In this case probably we should refuse to restore any token that has `introducerIdentity` at all, on the basis that we can no longer correctly enforce whatever it was trying to express. Or write a migration that deletes these tokens.

I vote for 2.